pkg/stores/common/test_db.go

```diff
@@ -18,14 +18,11 @@ import (
 
 var TestDBHost = os.Getenv("TEST_DB_HOST")
 var MigrationsPath = os.Getenv("DB_MIGRATIONS_PATH")
-
-const (
-	TestDBName     = "liwords_test"
-	TestDBPort     = "5432"
-	TestDBUser     = "postgres"
-	TestDBPassword = "pass"
-	TestDBSSLMode  = "disable"
-)
+var TestDBName = os.Getenv("DB_NAME")
```

Comment:
! this will delete your local database when running tests, please change this env var. `liwords` is the DB_NAME, the test db name should be `liwords_test` most likely.